community/duo_unix/APKBUILD

```diff
@@ -1,39 +1,38 @@
-# Contributor: Alan Lacerda <alacerda@alpinelinux.org>
-# Maintainer: Paul Morgan <jumanjiman@gmail.com>
+# Contributor: Kris Reese <ktreese@gmail.com>
+# Maintainer: Kris Reese <ktreese@gmail.com>
 pkgname=duo_unix
-pkgver=1.11.0
-pkgrel=1
-pkgdesc="duosecurity.com two-factor authentication"
-url="https://duo.com/support/documentation/duounix"
+pkgver=1.11.1
+pkgrel=0
+pkgdesc="Duo two-factor authentication for Unix systems with Pluggable Authentication Modules (PAM)"
+url="https://duo.com/docs/duounix"
 arch="all"
-license="GPL-2.0-only"
-depends=""
-depends_dev="openssl-dev zlib-dev"
-makedepends="$depends_dev automake autoconf libtool"
+license="GPL-2.0"
+depends="openssh openssh-server-pam"
+makedepends="openssl-dev linux-pam-dev openssh-server-pam libtool"
+install=""
+subpackages="$pkgname-doc"
+source="https://dl.duosecurity.com/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
 options="suid"
-subpackages="$pkgname-dev $pkgname-doc"
-source="https://github.com/duosecurity/duo_unix/archive/$pkgname-$pkgver.tar.gz"
-builddir="$srcdir"/$pkgname-$pkgname-$pkgver
 
 prepare() {
 	default_prepare
-	./bootstrap
 }
+
 build() {
 	cd "$builddir"
```

Comment:
Nowdays it can be omitted. abuild does it for you.

Reply:
Resolved via be68e705ba